setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='VacasaConnect',
-    version='4.4.3',
+    version='4.4.4',
```

Comment:
i think if we want to be real pedantic this'll be a `4.5.0` since it's adding functionality but backward compatible

Reply:
Good call.